src/records/d_record.rs

```diff
@@ -31,19 +30,25 @@ impl<'a> DRecord<'a> {
 
         let station_id = &line[2..6];
 
-        Ok(DRecord { qualifier, station_id })
+        Ok(DRecord {
```

Comment:
I'm not the greatest fan of using this many newlines - I think simple literals like this are more readable on a single line.
It looks like rustfmt is configurable enough to support this - I'll add a `rustfmt.toml` file at some point after merging this.

Reply:
Interesting, I wasn't aware yet that it was configurable :+1: